src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -310,11 +303,11 @@ def flexible_server_restart(cmd, client, resource_group_name, server_name, fail_
         raise ArgumentUsageError("Failing over can only be triggered for zone redundant servers.")
 
     if fail_over is not None:
-        if fail_over not in ['Planned', 'Forced']:
+        if fail_over not in ['Planned', 'Forced', 'planned', 'forced']:
```

Comment:
```suggestion
        if fail_over.lower() not in ['planned', 'forced']:
```
Just in case customer use `PLANNED` or `FORCED`

Reply:
good catch, thanks!!